### PR TITLE
refactor: Cleanup translations

### PR DESCRIPTION
**Pull request by Jonasdoubleyou**
Remove some unused components, move all texts to i18n.

### CODE REVIEW

File: src/widgets/certificates/SelectedPupilWizard.tsx [hunk 1]
```diff
@@ -93,7 +90,7 @@ const SelectedPupilWizard = ({
                 <VStack space={space['0.5']}>
                     <Text bold>Zeit</Text>
```
Comment (realmayus) [inline src/widgets/certificates/SelectedPupilWizard.tsx:91]: missing

File: src/widgets/SelectParticipants.tsx [hunk 2]
```diff
@@ -25,12 +27,12 @@ export function SelectParticipants({
                     {selectedParticipants.length !== 0 && selectedParticipants.map((it) => participants.find((p) => p.id === it)?.firstname).join(', ')}
```
Comment (realmayus) [inline src/widgets/SelectParticipants.tsx:27]: missing

**Review by realmayus — APPROVED** [review]
Nice!

**Review by realmayus — APPROVED** [review]

File: src/widgets/SelectParticipants.tsx (hunk 2 above)
Comment (Jonasdoubleyou) [reply src/widgets/SelectParticipants.tsx:27]: good catches :)

**Pull request merged**